assets/src/components/detours/diversionPage.tsx

```diff
@@ -164,11 +165,20 @@ export const DiversionPage = ({
   return (
     <>
       <article className="l-diversion-page h-100 border-box inherit-box">
-        <header className="l-diversion-page__header text-bg-light border-bottom">
+        <header
+          className={joinClasses([
+            "l-diversion-page__header",
+            "border-bottom",
+            snapshot.matches({ "Detour Drawing": "Active" }) &&
+            userInTestGroup(TestGroups.DetoursPilot)
+              ? "active-detour"
```

Comment:
looks like `--bs-bg-opacity` isn't set correctly so it's not showing up, but I think we should be able to use `text-bg-ui-alert` ?

Reply:
Ahhh I was wondering _what_ wasn't showing up, but I see that things look wonky on mobile. Is that what you're talking about? I'll fix it (but also the mobile view will be reworked further later)

Reply:
I was mainly thinking that we should be able to use the bootstrap classes for background colors and that when I tried it `text-bg-ui-alert` didn't work due to `--bs-bg-opacity` defaulting to completely transparent. I know `ui-alert` isn't lemon-100, so it's fine for now.